networkit/cpp/centrality/DynKatzCentrality.cpp

```diff
@@ -26,7 +26,7 @@ DynKatzCentrality::DynKatzCentrality(const Graph& G, count k, bool groupOnly, do
     });
     assert(maxdeg && "Alpha is chosen based on the max. degree; therefore, that degree must not be zero");
 
-    alpha = double(1)/(maxdeg + 1);
+    alpha = double(1)/ (double)(maxdeg + 1);
```

Comment:
Change `1` to `1.0` and the remaining functional cast to static cast.

Reply:
By changed the `1` to `1.0` the `static_cast` becomes redundant.